assets/src/data/ee-lists/index.js

```diff
@@ -0,0 +1,21 @@
+/**
+ * WordPress dependencies
+ */
+import { registerStore } from '@wordpress/data';
+
+/**
+ * Internal dependencies
+ */
+import reducers from './reducers';
+import * as selectors from './selectors';
+import * as actions from './actions';
+import * as resolvers from './resolvers';
+
+const store = registerStore( 'eventespresso/lists', {
```

Comment:
The intended purpose for the `eventespresso/lists` store is for any components that need to retrieve a list of EE entities for given queries for use in elements of that component.  The intent being that the retrieved entities are cached by that query so if there are multiple instances of that query requested in the same view, we avoid extra hits to the REST API endpoint for that query.